src/App.jsx

```diff
@@ -14,70 +16,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const getPrepareGoods = allGoods => allGoods
+  .map((good, index) => ({ good, key: index }));
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const prepareGoods = getPrepareGoods(goods);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export const App = () => {
+  const initialSelectedGood = prepareGoods[8].good;
```

Comment:
Can we move this variable outside the component to prevent unnecessary re-render